aliases.cwt

```diff
@@ -152,4 +152,21 @@ alias[command_modifier:narrow_flank] = float
 alias[command_modifier:winter_supply] = float
 alias[command_modifier:winter_combat] = float
 alias[command_modifier:actual_light_infantry] = float
-alias[command_modifier:actual_heavy_infantry] = float
\ No newline at end of file
+alias[command_modifier:actual_heavy_infantry] = float
+
+# Single aliases
+
+single_alias[general_triggers] = {
```

Comment:
Rename single_alias to `trigger_clause` and `effect_clause` please, think it's a bit clearer.

Reply:
Aye